queries.sql

```diff
@@ -1,10 +1,68 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = true;
-SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Update the animals table by setting the species column to 'digimon' for all animals that have a name ending in 'mon'
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to 'pokemon' for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+-- Verify that changes were made.
+SELECT * FROM animals;
+-- Commit the transaction.
+COMMIT;
+-- Verify that changes persist after commit.
+SELECT * FROM animals;
```

Comment:
- [ ] Before setting the species of the animals, kindly Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction. Also add screenshots of results you got from this to the pull request description as demanded by project requirements.